src/main/scala/Chisel/util/Decoupled.scala

```diff
@@ -22,21 +22,29 @@ object Decoupled {
   def apply[T <: Data](gen: T): DecoupledIO[T] = new DecoupledIO(gen)
 }
 
-/** An I/O bundle for enqueuing data with valid/ready handshaking */
+/** An I/O bundle for enqueuing data with valid/ready handshaking
```

Comment:
If this is really just a temporary fix, you should make that VERY CLEAR, so people won't rely on this behavior in their production code.